### PR TITLE
New Security Tools: Rainbow 6 Edition: Unfucked Edition

### PR DESCRIPTION
**Pull request by Nickvr628**
Ever conduct a security raid but realize the departmental officer didn't attend, and/or the AI is unavailable? Never again be locked out of the science lab preventing you from ~~executing~~ arresting the gang inside, with the new Tactical Breaching Shotgun!

image

This holds a brand new type of shotgun ammo, the Breacher Round, which can break most windows (but not grilles under them) instantly and shoot down airlocks in 3 shots. However, it only deals 10 damage to everything else, so using it on people would be pretty dumb. Consequently, this compact device only holds 3 rounds and only holds the breaching ammo. The ammo, however, can be loaded into any shotgun and can be found in a box in the armory or is producible inside the security protolathe for 4000 metal.

image
_edit: The shells look better now, scroll down for pictures_

More security tools are always good to add as it lets security explore additional options to deal with station challenges. Officers can take one of these to Reebe, for example, to help the crew by breaking down cult doors. They are also great when raiding departments as a group as it can break down welded or bolted doors. Alternatively, it can be used to quickly make window entry points to surprise defenders. 

The main drawback of this device is the low ammo capacity, and the fact that the ammo is expensive to make and only does 10 damage to anything not a door or window. 

This PR also includes RnD support for the included items for balance.

:cl:  
rscadd: The armory now contains breaching shotguns and specialized breaching ammo for them. Use them to quickly break down doors and create entry points, but they are nearly useless against people.
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
I pressed a lot of buttons and it went from 255 files changed to 14 changed, so I am happy again.

**Comment by Potototatoh** [comment]
poop slugs

**Comment by Nickvr628** [comment]
A lot of the ammo colors were already taken, I think the blue was as well. 

Also the pointed tip is supposed to be the oversized breaching slug that shows why it costs so much metal to make. It is staying.

**Comment by Ihonkification** [comment]
lighter brown please then.

pointed tip looks silly, make it like a slimmer cylinder or something

**Comment by Nickvr628** [comment]
A purple shell already exists but I liked your blunt nose slug design so I used it 
image
I will keep thinking about different colors, almost all are taken when you factor in the RnD shells.

**Comment by Ihonkification** [comment]
purple =/= pink

**Comment by Nickvr628** [comment]
image
After feedback and some consideration, I have changed the color scheme to be a blue shell with a brown stripe.

**Comment by Potototatoh** [comment]
thats better than poop slugs

**Comment by Xoxeyos** [comment]
May as well add tazers back after this gets merged.

**Comment by Nickvr628** [comment]
why?

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -797,6 +797,16 @@
 	for(var/i in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/buckshot(src)
 
+/obj/item/storage/box/breacherslug
+	name = "box of breaching shotgun shells"
+	desc = "A box full of breaching slugs, designed for rapid entry"
```
Comment (Amelia0010) [inline code/game/objects/items/storage/boxes.dm]:
```suggestion
	desc = "A box full of breaching slugs, designed for rapid entry."
```

File: code/modules/projectiles/ammunition/ballistic/shotgun.dm [hunk 2]
```diff
@@ -137,3 +137,11 @@
 	reagents.add_reagent(/datum/reagent/toxin/mutetoxin, 6) //;HELP OPS IN MAINT
 	reagents.add_reagent(/datum/reagent/toxin/coniine, 6)
 	reagents.add_reagent(/datum/reagent/toxin/sodium_thiopental, 6)
+
+/obj/item/ammo_casing/shotgun/breacher
+	name = "breaching slug"
+	desc = "A 12 gauge anti-material slug. Great for breaching airlocks and windows with minimal shots."
+	icon_state = "breacher"
+	caliber = "shotgun"
```
Comment (Amelia0010) [inline code/modules/projectiles/ammunition/ballistic/shotgun.dm]:
```suggestion
```

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 3]
```diff
@@ -40,6 +40,17 @@
 	sawn_desc = "Come with me if you want to live."
 	can_be_sawn_off  = TRUE
 
+// Breaching Shotgun //
+
+/obj/item/gun/ballistic/shotgun/automatic/breaching
+	name = "tactical breaching shotgun"
+	desc = "A compact semi-auto shotgun designed to fire breaching slugs and create rapid entry points"
```
Comment (Amelia0010) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]:
```suggestion
	desc = "A compact semi-auto shotgun designed to fire breaching slugs and create rapid entry points."
```

**Comment by Nickvr628** [comment]
Merge Ready TM

**Pull request merged**